src/themes/ec/maps/layout.scss

```diff
@@ -24,3 +24,4 @@ $grid-gutter-width: (
   'l': map.get($grid, 'gutter'),
   'xl': map.get($grid, 'gutter'),
 ) !default;
+$max-width: 580px !default;
```

Comment:
i find this a bit confusing, we could simply set this value in the custom_properties, then overriding the custom prop would change this for all its occurencies, if you prefer it like this, anyway, no issue.